contents/blog/product-health-metrics.md

```diff
@@ -0,0 +1,127 @@
+
+If growth is slowing, your product, positioning, or strategy might need to change. It is a sign:
+
+- your product isn’t appealing enough to your target users
+- you have the wrong features for your users
+- you aren’t targeting the right users
+
+Using this new user growth as a product metric helps you monitor and improve this.
+
+![New user growth](../images/blog/product-health-metrics/growth.png)
+
+### Churn rate
+
+[Churn rate](/blog/customer-churn-analysis-guide) is the rate users stop using the product. When a user churns, it means they haven’t returned to use the product recently. As a product health metric, churn lets you know whether users are leaving your product. 
+
+Churn rate is a key metric for many products and businesses. For SaaS or products relying on recurring usage (and revenue), churn might be the most important metric. This is because it is one of the key drivers of revenue and growth (along with new user growth).
+
+![Churn rate](../images/blog/product-health-metrics/churn.png)
+
+Some products have a naturally high churn, so having a high churn rate isn’t always a bad sign. A dramatic increase in churn is a bad sign though. It means something has changed significantly with your product and should inspire an intervention.
+
+### Daily, weekly, and monthly active users
+
+Between new user growth and churn rate is active users. This is a measure of users using your product calculated on a daily, weekly, or monthly basis.
+
+Consistent growth in active users means:
+
+- signups are continuing
+- you are keeping users engaged
+- users aren’t leaving
+
+Active users are the lifeblood of your product, so seeing them continue to grow means a product is healthy. Blood must continue to flow for your body to work, the same is true with your product.
+
+Doing a ratio of these values such as DAU/MAU can also be useful. This helps you understand how frequently people are engaging with the product. You can use PostHog’s formula mode to calculate this. 
+
+![DAU/MAU ratio](../images/blog/product-health-metrics/daumau.png)
+
+### Feature usage
+
+Having users is great, but you want them to actually use your product. Measuring feature usage helps you do this. This is tracking the usage of parts of your product, such as creating, analyzing, or sharing content.
+
+Feature usage helps you understand the health of individual parts of your product. Like the body, if you heal the "sick" parts, the whole is healthier. Feature usage improves prioritization by aligning development time with usage.
+
+Feature usage is a more specific health metric than new user growth or churn rate. It shows the health of specific features, which enables it to be more actionable.
```

Comment:
Could do with some examples of how to measure feature usage here. Would a stickiness or lifecycle insight make sense here?

Reply:
lifecycle is good